Content.Shared/Damage/Components/DamageableComponent.cs

```diff
@@ -112,61 +127,59 @@ public override void HandleComponentState(ComponentState? curState, ComponentSta
             }
         }
 
-        public int GetDamage(DamageType type)
+        public int GetDamage(DamageTypePrototype type)
         {
             return _damageList.GetValueOrDefault(type);
         }
 
-        public bool TryGetDamage(DamageType type, out int damage)
+        public bool TryGetDamage(DamageTypePrototype type, out int damage)
         {
             return _damageList.TryGetValue(type, out damage);
         }
 
-        public int GetDamage(DamageClass @class)
+        public int GetDamage(DamageGroupPrototype damageGroup)
```

Comment:
Rename the damageGroup argument to group to win instantly.

Reply:
me tiny brain no understand (also its been a month can you recap what i dun goofd here)

Reply:
oh wait i understand now i am no longer brain-damaged thanks